src/components/document-category-display.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Markdown from './markdown.js';
+import routes from '../utils/routes.js';
+import slugify from '@sindresorhus/slugify';
+import { useTranslation } from 'react-i18next';
+import { useService } from './container-context.js';
+import ClientConfig from '../bootstrap/client-config.js';
+import { getAccessibleUrl } from '../utils/source-utils.js';
+import { documentCategoryShape } from '../ui/default-prop-types.js';
+
+function DocumentCategoryDisplay({ documentCategory, bordered, detailed }) {
+  const { t } = useTranslation();
+  const clientConfig = useService(ClientConfig);
+
+  const iconUrl = documentCategory.iconUrl
+    ? getAccessibleUrl({ url: documentCategory.iconUrl, cdnRootUrl: clientConfig.cdnRootUrl })
+    : null;
+
+  const rootClasses = classNames({
+    'DocumentCategoryDisplay': true,
+    'DocumentCategoryDisplay--bordered': bordered,
+    'DocumentCategoryDisplay--detailed': detailed
+  });
+
+  const iconClasses = classNames({
+    'DocumentCategoryDisplay-icon': true,
```

Comment:
I found that with making the `DocumentCategoryDisplay-icon` `display: block`, it also aligns it vertically centered, like the text is. To me that looks better, at least with the icons I've tried so far.
<img width="446" alt="Screenshot 2024-04-26 at 09 41 35" src="https://github.com/educandu/educandu/assets/8189923/aff63662-f879-4f5f-984a-413dbffe7c81">
<img width="455" alt="Screenshot 2024-04-26 at 09 41 51" src="https://github.com/educandu/educandu/assets/8189923/65e28504-e49f-4972-8209-93556123d81f">

What do you think?

Reply:
👍